<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ccm_dialog</class>
 <widget class="QDialog" name="ccm_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CCM Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>208</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="OK">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>96</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="Cancel">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>228</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Configure  CCM  Matrix</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="9" column="0" colspan="6">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>ccm_coeff[2]:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QSpinBox" name="ccm_coeff5">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="minimum">
         <number>-5000</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="11" column="5">
       <widget class="QSpinBox" name="offset_out2">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="minimum">
         <number>-5000</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="11" column="4">
       <widget class="QLabel" name="label_12">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>offset_out[2]:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_13">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>9</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>ccm_coeff</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QSpinBox" name="ccm_coeff4">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="minimum">
         <number>-5000</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="value">
         <number>1024</number>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>offset_out</string>
        </property>
       </widget>
      </item>
      <item row="7" column="5">
       <widget class="QSpinBox" name="ccm_coeff8">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="minimum">
         <number>-5000</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="value">
         <number>1024</number>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>offset_out[0]:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>ccm_coeff[4]:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QSpinBox" name="ccm_coeff2">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="minimum">
         <number>-5000</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>ccm_coeff[1]:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="ccm_coeff3">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="minimum">
         <number>-5000</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QSpinBox" name="offset_out0">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="minimum">
         <number>-5000</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>ccm_coeff[6]:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="4">
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>ccm_coeff[8]:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="3">
       <widget class="QSpinBox" name="offset_out1">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="minimum">
         <number>-5000</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QSpinBox" name="ccm_coeff6">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="minimum">
         <number>-5000</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QSpinBox" name="ccm_coeff7">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="minimum">
         <number>-5000</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>ccm_coeff[7]:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="2">
       <widget class="QLabel" name="label_11">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>offset_out[1]:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="ccm_coeff0">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="minimum">
         <number>-5000</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="value">
         <number>1024</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>ccm_coeff[3]:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="ccm_coeff1">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="minimum">
         <number>-5000</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>ccm_coeff[5]:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>ccm_coeff[0]:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>OK</sender>
   <signal>clicked()</signal>
   <receiver>ccm_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Cancel</sender>
   <signal>clicked()</signal>
   <receiver>ccm_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>461</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
